orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/QueueShovel.kt

```diff
@@ -81,7 +90,33 @@ class QueueShovel(
     }
   }
 
+  private fun transferOwnership(message: Message) {
+    if (executionRepository == null) {
+      return
+    }
+
+    if (message !is ExecutionLevel) {
+      log.warn("Message $message does not implement ExecutionLevel, can not inspect partition")
+      return
+    }
+
+    // don't catch exceptions on retrieve/store (e.g. ExecutionNotFoundException), so that we can short-circuit shoveling
```

Comment:
is it worth catching for expected cases so we don't end up logging scary looking error logs?

Reply:
I see, so for instance we could avoid logging the stack trace for something like an `ExecutionNotFound` 👍